### PR TITLE
Release 0.2.29

### PR DESCRIPTION
**Pull request by johnc1231**
Change log for release 0.2.29. Currently I have the date set for tomorrow since i didn't know if it would make it in today. Could always change it.

### CODE REVIEW

File: hail/python/hail/docs/change_log.md [hunk 1]
```diff
@@ -24,6 +24,28 @@ an earlier version of Hail to read files written in a later version.
 
 ---
 
+## Version 0.2.29
+
+Released 2019-11-17
```
Comment (tpoterba) [inline hail/python/hail/docs/change_log.md]: 12-17

File: hail/python/hail/docs/change_log.md [hunk 2]
```diff
@@ -24,6 +24,28 @@ an earlier version of Hail to read files written in a later version.
...
+
+### Bug fixes
+- (hail#7229) Fixed `hl.maximal_independent_set` tie breaker functionality. See linked pull request for details.
```
Comment (tpoterba) [inline hail/python/hail/docs/change_log.md]: I think this "see for details" bit is implied, I think you can omit that sentence.

File: hail/python/hail/docs/change_log.md [hunk 3]
```diff
@@ -24,6 +24,28 @@ an earlier version of Hail to read files written in a later version.
...
+- (hail#7719) Generate more efficient IR for `Table.flatten`
+
+### New features
+- (hail#7686) Added `comment` argument to `import_matrix_table`, allowing lines with certain prefixes to be ignored.
+- (hail#7688) Added experimental support for `NDArrayExpression`s in new `hl.nd` module.
+- (hail#7608) `hl.grep` now has a `show` argument that allows users to either print the results (default) or return a dictionary of the results.
+
+### `hailctl dataproc`
+- (hail#7717) Throw error when mispelling arguments instead of silently quitting.
+
```
Comment (tpoterba) [inline hail/python/hail/docs/change_log.md]: missing a `-----` at the end I think

File: hail/python/hail/docs/change_log.md [hunk 4]
```diff
@@ -24,6 +24,28 @@ an earlier version of Hail to read files written in a later version.
...
+### Bug fixes
+- (hail#7229) Fixed `hl.maximal_independent_set` tie breaker functionality.
+- (hail#7732) Fixed incompatibility with old files leading to incorrect data read when filtering intervals after `read_matrix_table`.
+- (hail#7642) Fixed crash when constant-folding functions that throw errors.
+- (hail#7611) Fixed `hl.hadoop_ls` to handle glob patterns correctly.
+- (hail#7653) Fixed crash in `ld_prune` by unfiltering missing GTs. 
+
+### Performance improvements
+- (hail#7719) Generate more efficient IR for `Table.flatten`
```
Comment (tpoterba) [inline hail/python/hail/docs/change_log.md]: period at end (minor nit, but you'll need to edit for the horizontal bar anyway)

**Comment by johnc1231** [comment]
Also added Arcturus's method wrapping PR to changes.

**Pull request merged**